Description: This patch fixes spelling errors detected by the Debian tool lintian.
Author: Paul Gevers <elbrus@debian.org>
Bug: http://mantis.freepascal.org/view.php?id=30233

Index: fpc/install/man/man1/fp.1
===================================================================
--- fpc.orig/install/man/man1/fp.1
+++ fpc/install/man/man1/fp.1
@@ -49,7 +49,7 @@ on-line help system and a debugger.
 
 A text mode windowing system is the base of the user interface. The mouse is supported,
 but most people will use the keyboard. The user will usually open a few text editor
-windows in which he will design his program, during which he can regularily check and
+windows in which he will design his program, during which he can regularly check and
 debug through the use of hot keys. The high speed of the compiler ensures that programmers
 can do this often and therefore allows speedy development of code.
 
Index: fpc/install/man/man1/fpdoc.1
===================================================================
--- fpc.orig/install/man/man1/fpdoc.1
+++ fpc/install/man/man1/fpdoc.1
@@ -43,7 +43,7 @@ option.
 .BI \-\-cputarget=value
 This option tells 
 .B fpdoc
-for which CPU to generate documentation. This simply defines the approriate CPU macros.
+for which CPU to generate documentation. This simply defines the appropriate CPU macros.
 .TP
 .BI \-\-descr=descfile
 This option specifies the name of a description file
@@ -190,7 +190,7 @@ where to look for the translation files
 .BI \-\-ostarget=value
 This option tells 
 .B fpdoc
-for which operating system to generate documentation. This simply defines the approriate operating system macros.
+for which operating system to generate documentation. This simply defines the appropriate operating system macros.
 .TP
 .BI \-\-output=name 
 This option tells 
Index: fpc/install/man/man1/grab_vcsa.1
===================================================================
--- fpc.orig/install/man/man1/grab_vcsa.1
+++ fpc/install/man/man1/grab_vcsa.1
@@ -37,7 +37,7 @@ work). The permissions are automatically
 
 .B grab_vcsa
 is used by the Free Pascal video unit. The video unit allows programmers to
-develop portable programs accross Dos, Linux, FreeBSD, Windows, OS/2 and
+develop portable programs across Dos, Linux, FreeBSD, Windows, OS/2 and
 other operating systems while they can still assume being able to use the
 full VGA character set.
 
Index: fpc/install/man/man1/h2pas.1
===================================================================
--- fpc.orig/install/man/man1/h2pas.1
+++ fpc/install/man/man1/h2pas.1
@@ -101,7 +101,7 @@ macros are changed - wherever possible t
 are all integers, so these must be changed manually. Simple expressions 
 in define staments are recognized, as are most arithmetic operators: 
 addition, substraction, multiplication, division, logical operators, 
-comparision operators, shift operators. The C construct ( A ? B : C)
+comparison operators, shift operators. The C construct ( A ? B : C)
 is also recognized and translated to a pascal construct with an IF
 statement (this is buggy, however).
 
Index: fpc/install/man/man1/pas2jni.1
===================================================================
--- fpc.orig/install/man/man1/pas2jni.1
+++ fpc/install/man/man1/pas2jni.1
@@ -9,7 +9,7 @@ pas2jni - Free Pascal Java Native Interf
 Generate a Java native interface description and library from compiled Free Pascal units.
 pas2jni generates per unit 2 files: a library that provides access to the functions and 
 procedures in the pascal unit in a manner that Java understands, and a .java file that 
-describes the library, and can be used to access the libary from Java.
+describes the library, and can be used to access the library from Java.
 
 .SH OPTIONS
 .TP
Index: fpc/install/man/man1/ppdep.1
===================================================================
--- fpc.orig/install/man/man1/ppdep.1
+++ fpc/install/man/man1/ppdep.1
@@ -20,7 +20,7 @@ implementation section of the unit, and
 You can invoke 
 .B ppdep
 with as the only required argument the name of the
-file (program or unit) whose dependecies you wish to list. You don't need 
+file (program or unit) whose dependencies you wish to list. You don't need
 to specify an extension, by default 
 .B .pp
 is assumed.
